asyncpgsa/connection.py

```diff
@@ -92,18 +92,12 @@ def compile_query(query, dialect=_dialect, inline=False):
 
 
 class SAConnection(connection.Connection):
-    # __slots__ = ('_dialect')
     _dialect = None
 
-    # def __init__(self, connection_: connection, *, dialect=None):
-    #     self._connection = connection_
-    #     if not dialect:
-    #         dialect = _dialect
-    #     self._dialect = dialect
+    def __init__(self, *args, dialect=None, **kwargs):
+        super(SAConnection, self).__init__(*args, **kwargs)
 
-    # def __getattr__(self, attr, *args, **kwargs):
-    #     # getattr is only called when attr is NOT found
-    #     return getattr(self._connection, attr)
+        self._dialect = dialect if dialect is not None else _dialect
```

Comment:
This is fine, but I find `self._dialect = dialect or _dialect` a little more terse.